wire-ios-sync-engine/Tests/Source/Synchronization/SynchronizationMocks.swift

```diff
@@ -166,8 +166,8 @@ class FakeCredentialProvider: NSObject, ZMCredentialProvider {
     var email = "hello@example.com"
     var password = "verySafePassword"
 
-    func emailCredentials() -> ZMEmailCredentials! {
-        return ZMEmailCredentials(email: email, password: password)
+    func emailCredentials() -> UserEmailCredentials! {
```

Comment:
```suggestion
    func emailCredentials() -> UserEmailCredentials {
```
is that possible?

Reply:
Addressed it here: https://github.com/wireapp/wire-ios/commit/e93fc4bda3cdc961678f81f7980856357a3d3ef3